src/main/webapp/calendar-script.js

```diff
@@ -43,8 +66,95 @@ function getEvents(calendar) {
         title: event.name,
         start: event.strStartTime,
         end: event.strEndTime,
-        allDay: false
+        allDay: false,
+        extendedProps: {
+          stringDate: event.strStartTime,
+          address: event.address,
+          placeId: event.placeId, 
+          openTime: '9AM',
+          closeTime: '5PM',
+        }
       });
     });
   });
 }
+
+function createMap(modalBody, eventObj) {
+  const infoDisplay = document.createElement('div');
```

Comment:
This `div` (infoDisplay) isn't actually doing anything. Unless you add styling to this, you should take it out and add the `innerHTML` of `infoDisplay` directly to the `modalBody`.

Reply:
I need a div because the map is being added before the open hours can be

Reply:
i need it because i am adding the hours to the modalBody after i add the address and map. just the way it is structured. so if i want the open hours to be above the map I need the div

Reply:
OK gotcha, no problem 👍